packages/truffle-debugger/lib/session/sagas/index.js

```diff
@@ -98,7 +98,7 @@ function* fetchTx(txHash, provider) {
 
 function* recordContexts(...contexts) {
   for (let { contractName, binary, sourceMap } of contexts) {
-    yield *evm.addContext(contractName, binary);
+    yield *evm.addContext(contractName, { binary });
```

Comment:
Note: this is the first time a saga is defined that tags named arguments like this. I don't know if this break in pattern is a bad idea.

Reply:
Oh, interesting. Personally I'm fine with it, but I guess it's worth noting?  Anyway yeah all this passes my review at least.